### DISCUSSION TITLE
How to set pointer time format to 24-Hour

### DISCUSSION THREAD

**Discussion by chitanava** [vkurko/calendar#347] [Q&A] [answered]
Is there a way to change the date format for the pointer in the calendar? I have successfully changed the format for the slot labels using slotLabelFormat, but I cannot find an option to adjust the date format for the pointer. I would like it to be displayed in a 24-hour format.

![image](https://github.com/user-attachments/assets/3468eb76-511a-43ed-b748-97c04612eac9)

**Comment by emrahnalci** [comment]
You can use this code
```
slotLabelFormat: {hour: 'numeric', minute: '2-digit'},
eventTimeFormat: {hour: 'numeric', minute: '2-digit'},
dateClick: function(info){
console.log("dateClick Function");
console.log(new Date(info.date).toLocaleTimeString('en-GB', { hour: '2-digit', minute: '2-digit' }));
},
eventContent: function(info) {
 return {
 html: '<div class="ec-event-time">' + new Date(info.event.start).toLocaleTimeString('en-GB', { hour: '2-digit', minute: '2-digit' }) + ' - ' + new Date(info.event.end).toLocaleTimeString('en-GB', { hour: '2-digit', minute: '2-digit' }) + '</div>'
 };
},
```

**Comment by vkurko** [comment, marked as answer]
`pointer` uses the [eventTimeFormat](https://github.com/vkurko/calendar?tab=readme-ov-file#eventtimeformat) setting to format time.
